merkato/merkato.py

```diff
@@ -9,15 +9,55 @@
 
 
 class Merkato(object):
-    def __init__(self, configuration, ticker, spread):
+    def __init__(self, configuration, ticker, spread, ask_budget, bid_budget):
         self.exchange = Exchange(configuration)
         self.distribution_strategy = 1
         self.ticker = ticker # i.e. 'XMR_BTC'
         self.spread = spread # i.e '15
+        self.ask_profile = [(1, 5), (2, 5), (3, 10), (4, 15), (5, 20), (6, 25), (10, 30)] # (% change in price, % balance allocation)
```

Comment:
More of a MM question but why do the % balances add up to more than 100%, and what is the utility of having a non uniform ladder? I.e. allocating 1% of budget at each step of ladder.

Reply:
Bigger swings, bigger opportunity?

Maybe you want to hold 50% back for a 20% dip? Makes more sense with a flat pair, granted.

Reply:
@livinginformation  I figured the fui would have presets from you and a roll your own option

Reply:
Im still trying to figure out the best way to represent something that both needs to be able to span an infinite set (all potential prices), as well as not go over 100% allocation (geometric progression or something else). Self tuning is possible, but the representation is what im stuck on atm.

Reply:
The prices in this current iteration are infinite. So that is good.

 For allocation, error check would be a line or two, or a preprocess step that truncates the furthest away orders once allocation bis exhausted.

Reply:
@livinginformation "edge" use case: budget needs to be a bit smarter, I could see a case where you don't have any Pepecash but you wanna set a cap on budget assuming your bids get filled on the other half of the ladder.

Reply:
Is the solution to that case:  sell/buy does a balance check (in addition to market check)
This way the ladder loop would be best effort. @15chrjef @livinginformation